lib/Pipes/TaggedFunctionKind.cpp

```diff
@@ -66,3 +59,18 @@ void TaggedFK::getInvalidations(const Context &Ctx,
 
   // ToRemove.push_back(Target("0x1000", *this));
```

Comment:
Do not commit commented code!

Reply:
this was an example for antonio when  it started to work on invalidation. It is already gone in his branch

Reply:
Uh? I'm looking at this branch and it's going in first.